MeshLib/MeshEditing/AddLayerToMesh.cpp

```diff
@@ -31,6 +32,7 @@ namespace MeshLib
 
 MeshLib::Element* extrudeElement(std::vector<MeshLib::Node*> const& subsfc_nodes,
 	MeshLib::Element const& sfc_elem,
+	MeshLib::PropertyVector<int> const& sfc_to_subsfc_id_map,
```

Comment:
Can you document this method, even though it is in an anonymous namespace? The purpose can be guessed from the name, but not necessarily the purpose of the arguments.